samples/batch/transcription-enabled-storage/Connector/Serializable/TextAnalytics/TextAnalyticsInnerError.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="TextAnalyticsInnerError.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector.Serializable.TextAnalytics
+{
+    public class TextAnalyticsInnerError
```

Comment:
Why do you need a separate class TextAnalyticsInnerError which is almost the same  as TextAnalyticsError?

Reply:
Same as above: is used for deserialization/serialization.